src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -53,133 +56,131 @@ public void testEqualsHouse() {
         assertThat(structure.getType()).isEqualTo(StructureType.HOUSE);
```

Comment:
style

Must've missed this before, but the act here is actually an arrange. And the `structure.getType()` is actuall the act. You're testing the `getType()` method in this test.